deploy.sh

```diff
@@ -60,6 +60,7 @@ then
 	hugo -e production --minify
 	echo "Copying files for Netlify…"
 	cp _redirects public/_redirects
+    cp _headers public/_headers
```

Comment:
While this looks off here, this is actually "correct". And switched the tabs in the surrounding lines to spaces in a different PR. That's why I am also using spaces instead of tabs here.

Reply:
It looks like something broke here. Is the " missing intentionally?

Reply:
You broke that. :P 

https://github.com/datenanfragen/website/commit/6b55968593e46cc1b65dd5a5cdacc215393fac13#diff-60254338249f657a0a83f98258a56bfe